pkg/cfn/manager/cluster.go

```diff
@@ -18,7 +18,13 @@ func (c *StackCollection) CreateCluster(errs chan error, _ interface{}) error {
 		return err
 	}
 
-	return c.CreateStack(name, stack, c.spec.Metadata.Tags, nil, errs)
+	// All the tags contained in `c.spec.Metadata.Tags` are already contained in the shared tags.
+	// Providing it to CreateStack results in duplicated stack tags.
+	// See https://github.com/weaveworks/eksctl/issues/413 for more details.
+	tags := map[string]string{}
+
```

Comment:
Thanks for updating this, but please remove the lines above. For the future, no need to reference issues in comments.